packages/block/src/block.ts

```diff
@@ -366,13 +400,34 @@ export class Block {
     transactions: TypedTransaction[] = [],
     uncleHeaders: BlockHeader[] = [],
     withdrawals?: Withdrawal[],
+    executionWitness?: VerkleExecutionWitness | null,
```

Comment:
Ah, this needs to change for backwards compatibility reasons.

So for now we need to put `executionWitness?` after `opts`, as we did before on similar additions, even if not so beautiful.

Even if discouraged, there are always people (actually I guess: a lot) using the constructor directly and this would break things for them.

Reply:
so we would reorder on breaking changes? in that case whenever that happens i would prefer we pass here the {header,transactions,...} as the first arg rather than the args array

Reply:
Yes, I thought so too 😳. We should rearrange that along next breaking releases. This is otherwise always a bit annoying.

Reply:
Done